internal/mediainfo/mpeg_ts.go

```
@@ -2128,6 +2164,18 @@ func parseMPEGTSWithPacketSize(file io.ReadSeeker, size int64, packetSize int64,
 					delay := float64(int64(st.pts.min)-int64(videoPTS.min)) * 1000 / 90000.0
 					fields = append(fields, Field{Name: "Delay relative to video", Value: fmt.Sprintf("%d ms", int64(math.Round(delay)))})
 				}
+				if duration > 0 && st.audioBitRateKbps > 0 && size > 0 {
+					bps := int64(st.audioBitRateKbps) * 1000
```

golangci-lint (2.12.2)

[error] unnecessary conversion

(unconvert)